.travis.yml

```diff
@@ -2,9 +2,9 @@ language: csharp
 matrix:
   include:
     - os: linux
-      dist: trusty
+      dist: bionic
       mono: latest
-      dotnet: 2.1.3
+      dotnet: 3.0
```

Comment:
3.1 ?

Reply:
It seems 3.1 is not supported yet, but I will check.